src/v2/client/sessionStorageHelper.js

```diff
@@ -0,0 +1,29 @@
+/*!
+ * Copyright (c) 2021, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+const STATE_HANDLE_SESSION_STORAGE_KEY = 'osw-oie-state-handle';
```

Comment:
Out of curiosity, how did we decide on moving this to session storage versus another storage mechanism? Further, do we want to make this generic so developers can manage this themselves?

Time permitting, we should explore an additional configuration option to make this value (and which type of storage to use) overridable, similar with what we do with [okta-auth-js](https://github.com/okta/okta-auth-js/pull/266).

Reply:
we want stored stateHandle to be persistent only within the tab... not across the tabs.

https://oktawiki.atlassian.net/wiki/spaces/eng/pages/1185989427/Proposal+Limit+browser+token+storage+to+each+single+tab

Reply:
I think for this specific case, we want to `session` explicitly. @aaronbrodersen-okta  @pradeepdewda-okta can keep me honest.

cookie: we don't like it as it doesn't work cross the board and backend doesn't support it.
localStorage: will be shared cross tabs hence not ideal.